app/models/initializer_input.rb

```diff
@@ -0,0 +1,19 @@
+class InitializerInput
+  include Input::Common
+
+  ATTRIBUTES = [*Atlas::InitializerInput.attribute_set.map(&:name), :key]
+
+  attr_accessor *ATTRIBUTES
+
+  def self.get(key)
```

Comment:
Input and Iniitalizer input both define `.get` and `.fetch` and have identical implementations. Can they exist in `Input::Common` instead?

If you're finding that the `get` and `set` from InMemoryRecord are overwriting the ones from `Common`, you can instead include IMR into Common like so:

```ruby
class Input
  module Common
    extend ActiveSupport::Concern
    include InMemoryRecord

    included do
      include CommandAttributes
      include ActiveModel::Validations
    end

    # ...
  end
end
```

(including a module as part of the `included` block will include it *after* Common; including it as part of Common itself will do so *before* Common).

|    | `Input.ancestors` |
| --- | --- |
| Before | Input, ActiveModel::Validations::HelperMethods, ActiveSupport::Callbacks, ActiveModel::Validations, CommandAttributes, **InMemoryRecord, Input::Common**, ... |
| After | Input, ActiveModel::Validations::HelperMethods, ActiveSupport::Callbacks, ActiveModel::Validations, CommandAttributes, **Input::Common, InMemoryRecord**, ... |

In fact, you might prefer to do that for the other two modules as well?

```ruby
class Input
  module Common
    extend ActiveSupport::Concern
    extend ActiveModel::Naming

    include InMemoryRecord
    include CommandAttributes
    include ActiveModel::Validations

    included do
      attr_accessor :file_path
    end

    # ...
  end
end
```